### ISSUE TITLE
Where is L(mixco)?

### ISSUE THREAD

**Issue by Vincent950129** [Lee-Gihun/MixCo-Mixup-Contrast#1]
Hi,

I didn't see the L(mixco), could you please point me out which lines of code correspond to eq.3?

Thank you so much for your time.

**Comment by sungnyun** [comment]
Hi.

In the [loss_fn.py](https://github.com/Lee-Gihun/MixCo-Mixup-Contrast/blob/main/moco/utils/loss_fn.py), there is `MixcoLoss`.
If `mix_param` is not 0, it receives mixed logits and mixed labels (which are 'lambdas').

Thank you.
